plugin.video.botchamania/addon.xml

```diff
@@ -2,12 +2,15 @@
 <addon
 	id="plugin.video.botchamania"
 	name="Botchamania"
-	version="1.1.5"
+	version="1.1.6"
 	provider-name="Skipmode A1">
   <requires>
-    <import addon="xbmc.python"                 version="2.14.0"/>
-	<import addon="script.module.beautifulsoup" version="3.0.8"/>
-	<import addon="script.module.youtube.dl"    version="14.810.0"/>
+    <import addon="xbmc.python"                     version="2.14.0"/>
+	<import addon="script.module.beautifulsoup4"    version="4.5.3"/>
```

Comment:
@skipmodea1 ERROR: Version mismatch for addon script.module.beautifulsoup4. Required: 4.5.3, Available: 4.3.2